esphome/components/i2c/i2c_bus_arduino.cpp

```diff
@@ -45,22 +50,20 @@ void ArduinoI2CBus::dump_config() {
       break;
   }
   if (this->scan_) {
-    ESP_LOGI(TAG, "Scanning i2c bus for active devices...");
-    uint8_t found = 0;
-    for (uint8_t address = 8; address < 120; address++) {
-      auto err = writev(address, nullptr, 0);
-      if (err == ERROR_OK) {
-        ESP_LOGI(TAG, "Found i2c device at address 0x%02X", address);
-        found++;
-      } else if (err == ERROR_UNKNOWN) {
-        ESP_LOGI(TAG, "Unknown error at address 0x%02X", address);
-      }
-    }
-    if (found == 0) {
+    ESP_LOGI(TAG, "i2c bus scan results...");
```

Comment:
It's a bit awkward to start without a capital, and the `...` implies some action is being executed.
```suggestion
    ESP_LOGI(TAG, "Results from i2c bus scan:");
```